eventstream_rpc/include/aws/eventstreamrpc/EventStreamClient.h

```diff
@@ -608,7 +788,7 @@ namespace Aws
             }
 
             /**
-             * @return true if the instance is in a valid state, false otherwise.
+             * @return true if the connection is open, false otherwise.
```

Comment:
The original comment captures the intent of the API.  The new comment says what it actually does internally.  Not sure if this is an improvement or makes things less clear.

Reply:
This operator returns false for a closed connection or for a connection awaiting acknowledge, which are valid states. To me, the original comment feels misleading.

Reply:
Hmm, it's unfortunate that this is behaviorally different than every other bool cast operator in the CRT, but I supposed we can't change it now.

Reply:
The comment can be rephrased to something like this: `return true if the instance is valid and ready to send data, false otherwise`.